heartbeat/heartbeat.py

```diff
@@ -23,5 +24,15 @@
         print("Variable TEMPLATE_ID is missing")
     for base_url in BASE_URL:
         notifications_client = NotificationsAPIClient(API_KEY, base_url=base_url)
-        response = notifications_client.send_email_notification(email_address=EMAIL_ADDRESS, template_id=TEMPLATE_ID)
-        print("Email has been sent!")
+        try:
+            response = notifications_client.send_email_notification(email_address=EMAIL_ADDRESS, template_id=TEMPLATE_ID)
+            print("Email has been sent by {}!".format(base_url))
+        except HTTPError as e:
+            print("There was an error")
+            print(e.status_code)
+            print(e._message)
```

Comment:
Can we provide a bit more clarity on the error msg?
```
print(f"Could not send heartbeat: status={e.status_code}, msg={e._message}")
```

Reply:
done